tests/integration/galley/validation_test.go

```diff
@@ -123,6 +123,16 @@ func TestValidation(t *testing.T) {
 							t.Fatalf("config request denied for wrong reason: %v", err)
 						}
 					}
+
+					wetRunErr := env.ApplyContents(ns.Name(), ym)
+					defer func() { _ = env.DeleteContents(ns.Name(), ym) }()
+
+					if err != nil && wetRunErr == nil {
```

Comment:
is it even possible for `err != nil` at this point, due to the `t.Fatal` calls above?

Reply:
yes. if there is an error, the testdata is not valid and the error string is one of the recognized k8s api server errors, the error still exists. Here we want to test the dryrun and wetrun both return errors or neither return errors.